src/marqo/tensor_search/tensor_search.py

```diff
@@ -1156,7 +1158,7 @@ def _lexical_search(
     """
     if not isinstance(text, str):
         raise errors.InvalidArgError(
```

Comment:
Will this error ever be raised with the pedantic validation?

Reply:
Yes, because `_lexical_search` never actually uses pydantic validation. This validation happens in the `SearchQuery` api model, which is only used by `_vector_text_search`. We should definitely change this in the future.

Reply:
Got it!